implementations/elixir/ockam/ockam_healthcheck/test/healthcheck/healthcheck_test.exs

```diff
@@ -115,7 +115,7 @@ defmodule Ockam.Healthcheck.Test do
       :telemetry.detach("test_handler")
     end)
 
-    assert {:error, :timeout} = Ockam.Healthcheck.check_target(target, 1000)
+    assert {:error, :timeout} = Ockam.Healthcheck.check_target(target, :ping_target, 1000)
```

Comment:
Since we already have `check_endpoint` as a separate function, can we keep this API as is and just use a separate one for api targets, instead of introducing an argument?